oe_editorial.module

```diff
@@ -6,3 +6,52 @@
  */
 
 declare(strict_types = 1);
+
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\Url;
+
+/**
+ * Implements hook_field_widget_form_alter().
+ */
+function oe_editorial_field_widget_form_alter(&$element, FormStateInterface $form_state, $context) {
+  /** @var \Drupal\Core\Field\FieldDefinitionInterface $field_definition */
+  $field_definition = $context['items']->getFieldDefinition();
+
+  // We add an after build method so we can alter the Url of the help link.
+  if (in_array($field_definition->getType(), [
```

Comment:
Let's try to target all the possible "text_format" elements by adding a process function there.